05 - HTML/README.md

```diff
@@ -1 +1,437 @@
 # HTML
+
+All websites you visit daily, even some apps for your smartphone are created with HTML. What's HTML? The definition is Hyper Text Markup Language. Remember when we talked about Hyper Text on the How The Internet Works topic? Well, here is were things start to makes sense.
+
+The word Markup comes from the expresion to "Mark up" some manuscript, and that annotations were added to the document in order to add some styles, like changing the font, adding color, changing size of the text, etc.
+
+Let's leave something clear first: HTML it's not a programming language, HTML it's just a Markup Language. This is because HTML is not able to handle logic, you can't create reusable functions neither create variables to storage data, you can't have also control structures, this language allows us to give the structure of a document which will give life to our websites.
+
+As we said in previous topics, all in the internet are files. At the end, every website is an html file or a group of files, so if you want to start creating your own, open your favorite text editor and create a file with extension `.html`. For convention, the main page of your site should be called `index.html`.
+
+HTML uses tags in order to markup text and create elements this way. A tag, generally has a beginning and an ending, and receive the name of *Paired tags*. This is the syntax for the root tag:
+
+```
+<html></html>
+```
+
+We must have clear that not all the tags will have the same structure, there are other tags where the element are not going to have end, just because they are not marking up any text or other elements, these are only elements that will be inserted into the page and tags receive the name of *Single tags*. The syntax of these are similar to this one:
```

Comment:
`where the element are not going to have end`

it should be

`where the elementS are not going to have end`

or better:

`there are other elements that don't have closing tags`